builder.go

```diff
@@ -68,6 +69,29 @@ func NewBuilder(root ed25519.PrivateKey, opts ...builderOption) Builder {
 	return b
 }
 
+func (b *builder) AddBlock(block ParsedBlock) error {
+	for _, f := range block.Facts {
+		err := b.AddAuthorityFact(f)
+		if err != nil {
+			return err
+		}
```

Comment:
these can be a little cleaner like this:
```suggestion
		if err := b.AddAuthorityFact(f); err != nil {
			return err
		}
```